src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Server/ServerHandler.cs

```diff
@@ -75,38 +74,50 @@ public override Task<InitializeViewResult> InitializeObjectView(InitializeViewRe
 
         public override Task Save(ServerViewContext context, ServerInfo obj)
         {
-            UpdateServerProperties(context.Parameters, obj);
+            UpdateServerProperties(context.Parameters, obj, RunType.RunNow);
             return Task.CompletedTask;
         }
 
         public override Task<string> Script(ServerViewContext context, ServerInfo obj)
         {
-            throw new NotSupportedException("ServerHandler does not support Script method");
+            var script = UpdateServerProperties(
+                 context.Parameters,
+                 obj,
+                 RunType.ScriptToWindow);
+            return Task.FromResult(script);
         }
 
-        private void UpdateServerProperties(InitializeViewRequestParams viewParams, ServerInfo serverInfo)
+        private string UpdateServerProperties(InitializeViewRequestParams viewParams, ServerInfo serverInfo, RunType runType)
         {
-            if (viewParams != null)
+            if (viewParams == null)
```

Comment:
This if statement is empty, so is it still needed?

Reply:
Oops I forgot to add logging here, but now that I'm thinking about it maybe we don't actually need this extra check